Tensile/TensileCreateLibrary.py

```diff
@@ -518,53 +523,45 @@ def buildKernelSourceAndHeaderFiles(results, outputPath):
         kernelsToWrite.append((err, src, header, kernelName))
```

Comment:
Seems like we are recreating `results` aside from dropping filename.

Reply:
And we're appending to two objects, one of which we only use to check that it's not null. I'll remove it.